src/plus/webviews/graph/graphWebview.ts

```diff
@@ -2395,6 +2433,11 @@ export interface GraphTagContextValue {
 	ref: GitTagReference;
 }
 
+export interface GraphRevisionContextValue {
```

Comment:
I don't think we need both `GraphCommitContextValue` and `GraphRevisionContextValue`, can we reuse `GraphCommitContextValue` (since internal GL handles WIP as the "uncommitted" commit) or if we just want to go with revision rename it to `GraphRevisionContextValue`?

If we go with revision, we'd just need to update the usages of the context where we set the `type` of the context to `commit` and change it to `revision` (which does align with how we call it internally)

Reply:
Decided to just use `GraphCommitContextValue`. Thanks!

Done in commit: [Use commit context value](https://github.com/gitkraken/vscode-gitlens/pull/2458/commits/c081cdd5267c362eb7e49cba287a2f10bb81b648)